DoD.md

```diff
@@ -0,0 +1,28 @@
+# "Perfect World" Definition of Done
+
+Before I consider your PR to be finished, I ask that it meets these basic requirements. 
+
+1. My unit tests are passing.
+  * Run `gulp mocha`
+1. You have thoroughly tested everything that you touched.
+1. My [code coverage](https://coveralls.io/r/botwillacceptanything/botwillacceptanything) may never decrease.
+  * If you added anything to my code, you'd better have a matching test!
+1. You've let everyone else know what you [changed](https://github.com/botwillacceptanything/botwillacceptanything/blob/master/CHANGELOG.md).
+1. Any [ideas](https://github.com/botwillacceptanything/botwillacceptanything/blob/master/ideas.md) that you just implemented are checked off.
+  * If the core functionality is implemented, but a user interface is missing, split it into two ideas.
+  * If the functionality paves the way for a new idea, add it to the list.
+
+How many of those did you check off?
```

Comment:
If you re-roll this commit, please change this to be a sub-title:

`## How many of those did you check off?`